src/bundle/Resources/public/js/scripts/core/adaptive.items.js

```diff
@@ -2,14 +2,18 @@
     const OFFSET_ROUNDING_COMPENSATOR = 0.5;
     class AdaptiveItems {
         constructor(config) {
+            this.isVertical = config.isVertical ?? false;
             this.container = config.container;
             this.items =
                 config.items ||
                 this.container.querySelectorAll(':scope > .ibexa-adaptive-items__item:not(.ibexa-adaptive-items__item--selector)');
-            this.selectorItem = config.selectorItem || this.container.querySelector(':scope > .ibexa-adaptive-items__item--selector');
+            this.selectorItem = config.selectorItem ?? this.container.querySelector(':scope > .ibexa-adaptive-items__item--selector');
             this.itemHiddenClass = config.itemHiddenClass;
             this.getActiveItem = config.getActiveItem;
             this.onAdapted = config.onAdapted;
+            this.itemOffset = config.itemOffset ?? 0;
+            this.classForceHide = config.classForceHide ?? 'ibexa-adaptive-items__item--force-hide';
+            this.classForceVisible = config.classForceVisible ?? 'ibexa-adaptive-items__item--force-visible';
```

Comment:
nitpick: maybe
```suggestion
            this.classForceVisible = config.classForceVisible ?? 'ibexa-adaptive-items__item--force-show';
```
as `hide` is already a verb.